hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/util/OS.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.tpc.util;
+
+/**
+ * Utility methods for OS specific functionality.
+ */
+public class OS {
```

Comment:
I assume some of the unused methods here are for the io-uring, right? So they will be used in the future?

Reply:
Correct.